src/main/webapp/js/planner.js

```diff
@@ -78,7 +80,7 @@ function createTable(tableData, courseContainer) {
     const row = document.createElement('tr');
     const cell = document.createElement('td');
     const semesterLabel = document.createElement('b');
-    semesterLabel.innerText = `Semester ${i + 1}:`;
+    semesterLabel.innerText = `Semester ${i + 1} (${creditsData[i]} Credits):`;
```

Comment:
Maybe add a check that tableData and creditsData have the same size, to make the code more robust.